go-controller/pkg/node/egressip.go

```diff
@@ -192,13 +192,15 @@ func (e *egressIPLocal) notifyClusterNodes(egressIP string) {
 		klog.Errorf("Unable to get link for primary interface: %s, err: %v", e.defaultGatewayIntf, err)
 		return
 	}
-	out, err := exec.Command("/sbin/arping", "-q", "-A", "-c", "1", "-I", primaryLink.Attrs().Name, egressIP).CombinedOutput()
+	_, stderr, err := util.RunArping("-q", "-A", "-c", "1", "-I", primaryLink.Attrs().Name, egressIP)
 	if err != nil {
-		klog.Warningf("Failed to send ARP claim for egress IP %q: %v (%s)", egressIP, err, string(out))
+		klog.Warningf("Failed to send ARP claim for egress IP %q: %v (%s)", egressIP, err, string(stderr))
 		return
 	}
 	time.Sleep(2 * time.Second)
-	_ = exec.Command("/sbin/arping", "-q", "-U", "-c", "1", "-I", primaryLink.Attrs().Name, egressIP).Run()
+	if _, stderr, err := util.RunArping("-q", "-U", "-c", "1", "-I", primaryLink.Attrs().Name, egressIP); err != nil {
```

Comment:
what is the goal here? It would be one thing if you were creating an abstraction to hide the `arping` command-line args or something, but just having a thin wrapper around `exec.Command` doesn't seem like an improvement?

Reply:
The goal is being able to use our testing package (`fakeExec`) and verify that the proper `arping` commands are being executed.